tools/config-generator/udpaterelease.go

```diff
@@ -71,6 +72,13 @@ func getReposMap(val interface{}, latest string) interface{} {
 			skipCiUpdate      bool
 			skipReleaseUpdate bool
 		)
+		repoName := getString(repo.Key)
+		latest, err := latestReleaseBranch(gc, repoName)
+		if err != nil {
+			return nil, fmt.Errorf("failed getting latest release branches: %w", err)
+		}
+
+		log.Printf("latest branch for repo %q is %q", repoName, latest)
```

Comment:
Nit: capitalize the first word in the logging message

Reply:
done